roles/thelounge/templates/config.js.j2

```diff
@@ -0,0 +1,472 @@
+    // (controlled by `maxHistory`), and the folder is cleaned up when The Lounge
+    // restarts.
+    //
+    // This value is set to `false` by default.
+    prefetchStorage: {{ thelounge__prefetch_storage | to_json }},
+    // ### `prefetchMaxImageSize`
+    //
+    // When `prefetch` is enabled, images will only be displayed if their file
+    // size does not exceed this limit.
+    //
+    // This value is set to `2048` kilobytes by default.
+    prefetchMaxImageSize: 2048,
+    // ### prefetchMaxSearchSize
+    //
+    // This value sets the maximum response size allowed when finding the Open
+    // Graph tags for link previews. The entire response is temporarily stored
+    // in memory and for some sites like YouTube this can easily exceed 300
+    // kilobytes.
+    //
+    // This value is set to `50` kilobytes by default.
+    prefetchMaxSearchSize: 50,
+    // ### `prefetchTimeout`
+    //
+    // When `prefetch` is enabled, this value sets the number of milliseconds
+    // before The Lounge gives up attempting to fetch a link. This can be useful
+    // if you've increased the `prefetchMaxImageSize`.
+    //
+    // Take caution, however, that an inordinately large value may lead to
+    // performance issues or even a denial of service, since The Lounge will not
+    // be able to clean up outgoing connections as quickly. Usually the default
+    // value is appropriate, so only change it if necessary.
+    //
+    // This value is set to `5000` milliseconds by default.
+    prefetchTimeout: 5000,
+    // ### `fileUpload`
+    //
+    // Allow uploading files to the server hosting The Lounge.
+    //
+    // Files are stored in the `${THELOUNGE_HOME}/uploads` folder, do not expire,
+    // and are not removed by The Lounge. This may cause issues depending on your
+    // hardware, for example in terms of disk usage.
+    //
+    // The available keys for the `fileUpload` object are:
+    //
+    // - `enable`: When set to `true`, files can be uploaded on the client with a
+    //   drag-and-drop or using the upload dialog.
+    // - `maxFileSize`: When file upload is enabled, users sending files above
+    //   this limit will be prompted with an error message in their browser. A value of
+    //   `-1` disables the file size limit and allows files of any size. **Use at
+    //   your own risk.** This value is set to `10240` kilobytes by default.
+    // - `baseUrl`: If you want to change the URL where uploaded files are accessed,
+    //   you can set this option to `"https://example.com/folder/"` and the final URL
+    //   would look like `"https://example.com/folder/aabbccddeeff1234/name.png"`.
+    //   If you use this option, you must have a reverse proxy configured,
+    //   to correctly proxy the uploads URLs back to The Lounge.
+    //   This value is set to `null` by default.
+    fileUpload: {
+        enable: false,
+        maxFileSize: 10240,
+        baseUrl: null,
+    },
```

Comment:
This is a very handy feature which we are already using in assek, so it would be nice to be able to enable it

Reply:
Ops, I missed it, I am going to enable it.

Reply:
I done it via https://github.com/Tina-pm/ansible-roles/compare/1631caaed6850c71ce8024013eea7a6b81c72901..6e50d632e797aece7e05f67fb2366d01f849650e and https://github.com/Tina-pm/sysadmin/compare/514fe9cc4bbaf35e4d4cecdacb848233752c3b7d..577570834bfd7bffd23a42afc4090c12e1807e8b